packaging/debian-sid/changelog

```diff
@@ -1,3 +1,11 @@
+snapd (2.51.6-1) unstable; urgency=medium
+
+  * New upstream release, LP: #1929842
+    - secboot: use half the mem for KDF in AddRecoveryKey
```

Comment:
In retrospect I should have edited this to describe a bit better what it really does